cms/check_rule_counts

```diff
@@ -50,61 +86,120 @@ if __name__ == '__main__':
                      'SUSPENDED': 'rules_suspend',
                      'WAITING_APPROVAL': 'rules_waiting_approval', }
 
+        # I think both dictionaries in this part of the script are less confusing
+        ages = {
+            'created_24hours_ago': datetime.timedelta(days=1),
+            'created_1week_ago': datetime.timedelta(days=7),
+            'created_3weeks_ago': datetime.timedelta(days=21),
+        }
+
         result = (session.query(models.ReplicationRule.state, func.count(models.ReplicationRule.state))
                   .group_by(models.ReplicationRule.state)
                   .with_hint(models.ReplicationRule, 'INDEX_FFS(rules RULES_PK)', 'oracle')
                   .all())
 
+        print(result)
         for state, num in result:
+
+    # Count Rules in x state
             gauge_state = state_map.get(repr(state), 'rules_' + repr(state).lower())
```

Comment:
this is much easier. repr(state) can be replaced by str(state.name) and then no substitutions in the next line.